db/migrate/20170512074659_deduplicate_messages_for_arf_reports.rb

```diff
@@ -0,0 +1,27 @@
+class DeduplicateMessagesForArfReports < ActiveRecord::Migration
+  def up
+    ForemanOpenscap::Policy.unscoped.all.find_in_batches do |batch|
+      batch.each do |policy|
+        process_policy policy
+      end
+    end
+  end
+
+  def process_policy(policy)
+    reports_of_policy = ForemanOpenscap::ArfReport.unscoped.of_policy(policy)
+    latest = reports_of_policy.first
+    reports_of_policy.each do |report|
+      next if report == latest
+      report.logs.each do |log|
+        latest_log = latest.logs.find_by(:source_id => log.source_id)
+        next unless latest_log
+        next if log == latest_log
+        latest_message = latest_log.message
+        msg = log.message
+        log.message = latest_message
+        log.save!
```

Comment:
`log.update_attribute` would be probably safer in case log would have some validations added later, other options is to call save with `:validate => false`